### DISCUSSION TITLE
Palworld Console Command Not Starting

### DISCUSSION THREAD

**Discussion by chucklefarts** [GameServerManagers/LinuxGSM#4475] [Q&A] [answered]
So, my server seems to be working fine but I would like to use the console to see who is online.  When I try using `./pwserver console` I get the following error:

```
Shutdown handler: initalize.
Increasing per-process limit of core file size to infinity.
dlopen failed trying to load:
steamclient.so
with error:
steamclient.so: cannot open shared object file: No such file or directory
[S_API] SteamAPI_Init(): Loaded '/home/pwserver/.steam/sdk64/steamclient.so' OK.  (First tried local 'steamclient.so')
CAppInfoCacheReadFromDiskThread took 5 milliseconds to initialize
dlmopen steamservice.so failed: steamservice.so: cannot open shared object file: No such file or directory
Setting breakpad minidump AppID = 2394010
[S_API FAIL] Tried to access Steam interface SteamUser021 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface SteamFriends017 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface STEAMAPPS_INTERFACE_VERSION008 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface SteamNetworkingUtils004 before SteamAPI_Init succeeded.
```

It freezes after that.  I don't really know what is causing this but I found something about the steamservice.so file being forgotten to be packaged into the latest steam update: https://github.com/ValveSoftware/steam-for-linux/issues/10393

Does anyone else have this issue?  Basically I want to know if this is a "me" problem and everyone else can use the command fine of if everyone is also having this problem.  If anyone knows how I could fix this please let me know.  Thank you!

**Comment by lblanchardiii** [comment]
Mine shows the same when I try to access the console.

**Reply by chucklefarts** [reply]
So, I suppose this is an issue on Valve's end.  Does anyone know how to update steamcmd if/when this is fixed?  Thanks for the help.

**Reply by nicanordlc** [reply]
Having the same issue, Thanks

**Comment by tjrjaws** [comment]
Same issue.

**Comment by dgibbs64** [comment, marked as answer]
The console command allows access the output of the game server executable. Depending on which server you are running it allows you to see the current activity of the game server, check errors in real time, and input commands. Different game servers can be anywhere between interactive and verbose outputs to non-interactive and not verbose. Most backend consoles do not typically output player numbers. In other words the console is outputting as expected. You can however install gamedig to see player numbers from the details command. or even try out my new project https://ismygameserver.online 

https://docs.linuxgsm.com/commands/console
